src/loaders.ts

```diff
@@ -602,6 +601,68 @@ export type BlockscoutContractResult = {
     has_methods_write_proxy?: boolean;
 };
 
+
+function isAnyABINotFound(error: any): boolean {
```

Comment:
Starting to think we need an `ABILoader.isNotFound(error:any): boolean` that we can inherit/override, but maybe premature?